File: app/src/main/java/com/devzamse/android/tallertemple/MainActivity.java
package com.devzamse.android.tallertemple;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText usuario, contrasena, celular;
    Button login;
    String user, pass;
    TextView hack;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        usuario = findViewById(R.id.usuario);
        contrasena = findViewById(R.id.contrasena);
        celular = findViewById(R.id.celular);
        login = findViewById(R.id.login);
        hack = findViewById(R.id.hack);

        ClickLogin();

    }

    private void ClickLogin() {
        login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                user = usuario.getText().toString();
                pass = contrasena.getText().toString();

                if(user.equals("temple1") && pass.equals("abc") && celular.getText().toString().equals("1")){
                    Intent a = new Intent(MainActivity.this, broma.class);
                    startActivity(a);
                } else {
                    hack.setText(user+" "+pass);
                    Toast.makeText(MainActivity.this, "Dejame hackear", Toast.LENGTH_SHORT).show();
                }

//                Toast.makeText(MainActivity.this, user+" "+pass , Toast.LENGTH_SHORT).show();
                Log.e("Usuario: ", user);
                Log.e("Pass: ", pass);
                Log.e("Celular: ", celular.getText().toString());
            }
        });
    }
}

File: kotlin/app/src/main/java/com/devzamse/android/kotlin_temple/MainActivity.kt
package com.devzamse.android.kotlin_temple

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Window
import android.view.WindowManager
import androidx.core.content.ContextCompat
import com.devzamse.android.kotlin_temple.view.menu.inicio

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        ConfigInicial()

    }

    private fun ConfigInicial() {
        val window: Window = window
        window.clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS)
        window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
        window.statusBarColor = ContextCompat.getColor(baseContext, R.color.rojo)
        window.navigationBarColor = ContextCompat.getColor(baseContext, R.color.rojo)

        supportFragmentManager.beginTransaction().replace(R.id.frame, inicio()).commit()
    }
}
